app/assets/stylesheets/_buttons.scss

```diff
@@ -69,6 +68,46 @@ a .button,
 .orange_submit_button { padding: 5px; }
 .translation_button { @extend .play_video_button; }
 
+.buttons-container {
+  bottom: 0;
+  display: flex;
+  justify-content: space-between;
+  left: 0;
+  text-align: left;
+  width: 100%;
+  .add-to-vocab-btn {
+    display: flex;
+  }
+}
+
+.download-link {
+  width: 50%;
+}
+
+.download-button,
+.add-button-icon {
+  bottom: -4px;
+  height: 2.0625rem;
+  margin: 0 2px 0 0;
+  position: relative;
+  width: 2.1875rem;
+}
+
+.add-button-text {
```

Comment:
You can delete this add-button-text class if you also remove the float-right property thats been assigned on line 58 of _search_result.scss below

Reply:
@Dan I have removed the right float. All ok then when removing the 2.5rem left padding means the text floats over the button.